sympy/geometry/ellipse.py

```diff
@@ -1005,6 +1005,37 @@ def semilatus_rectum(self):
         """
         return self.major * (1 - self.eccentricity ** 2)
 
+    def auxiliary_circle(self):
+        """Returns equation of auxiliary circle of the ellipse.
```

Comment:
This needs to be updated to reflect that it is not returning an equation.

Reply:
I have updated the return docstring is it good

Reply:
I have updated the return docstring is it good @smichr